munich-university-of-applied-sciences-department-of-applied-social-sciences.csl

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+  <info>
+    <title>Munich University of Applied Sciences - Department of Applied Social Sciences</title>
```

Comment:
@POBrien333, for non-English styles, we usually use a title in the language of the style (so `<title>Hochschule München - Fakultät für Angewandte Sozialwissenschaften (German)</title>` here, with the language indicated in English).

Reply:
(and drop the umlauts for the style ID, i.e. "Fakultät für" becomes "fakultat-fur")